.travis.yml

```diff
@@ -23,4 +23,4 @@ env:
   # SONA_PASS
```

Comment:
Remove before release.

Reply:
Although it looks like this might not yet be working as expected.

Reply:
Why this should be removed?

Reply:
This is a temporary commit that disables the publication of docs. Apparently, it was not working as expected and so had to be disabled. We need to enable it before release.